spec/handlers/newflow/oauth_callback_spec.rb

```diff
@@ -73,5 +73,18 @@ module Newflow
         end
       end
     end
+
+    context 'when authentication found, but the response from social provider does not include email' do
+      let(:request) {
+        create_newflow_user(email)
```

Comment:
these seem like things that should be split into lets and before do/end blocks?  the info is a let, create_newflow_user is a before, the request is the subject (object being tested)?  separating like this creates clarity, and clarity improves readability and maintainability ... 

http://www.betterspecs.org/#subject

Reply:
Will do. Great suggestions! 🙏

Reply:
Done.